app/views/content/improve-your-subject-knowledge.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Improve your subject knowledge
+image: "/assets/images/international-dt.jpg"
+article_classes: ['longform']
+mobileimage: "/assets/images/steps-hero-mob.jpg"
+description: |-
+  School experience can help you talk to teachers and get an insight into day-to-day school life. Here's how to get school experience.
+date: "2021-05-04"
+right_column:
+  ctas:
+    - title: Find and apply for teacher training
+      link_text: "Find postgraduate teacher training courses"
+      link_target: "https://www.gov.uk/find-postgraduate-teacher-training-courses"
+      icon: "icon-arrow"
+      hide_on_mobile: Yes
+      hide_on_tablet: Yes
+keywords:
+  - Subject knowledge
+  - Subject knowledge enhancement
+  - SKE
+---
+If you're concerned about how well you know the subject you want to teach, do not worry - you may be able to do a 'subject knowledge enhancement' course.
+
+When you apply for teacher training, the school or university will help you establish whether you need to improve your subject knowledge.
+
+They may ask you to do a course if you have:
+
+* a different but related degree to the subject you want to teach
+* an A level but not a degree in the subject
+* an unrelated degree but relevant professional experience
+* a languages degree but need to improve your knowledge of a language you’ll teach
+* not used your subject knowledge for a while
+
+Completing a course may be a condition of your offer to train.
+
+[Check whether there’s a subject knowledge enhancement course for your subject](https://www.gov.uk/government/publications/subject-knowledge-enhancement-course-directory/subject-knowledge-enhancement-ske-course-directory). They're typically offered for subjects where there’s a particular demand for teachers.
```

Comment:
I wonder if a future enhancement might be making this a specific tiled CTA at the bottom of the page as a reasonable next step.

Reply:
yea - although I almost didn't want to make a big deal of this call to action - it's more of 'have a quick look' then get on with applying thing I was going for!